src/Http/Routing/src/Builder/RoutingEndpointConventionBuilderExtensions.cs

```diff
@@ -115,7 +115,8 @@ public static TBuilder WithName<TBuilder>(this TBuilder builder, string endpoint
     /// Sets the <see cref="EndpointGroupNameAttribute"/> for all endpoints produced
```

Comment:
This also seems confusing -- what does it mean to "set an attribute"?

Maybe this should say "Sets the IEndpointGroupNameMetadata" ? or ???

Reply:
Yeah, I think this doc comment is attempting to apply a literal description of what this method is doing. It instantiates the `EndpointGroupNameAttribute` class and adds it to the metadata collection.

However, that's a means to an end for the real valuable thing which is the fact that `EndpointGroupNameAttribute` implements the `IEndpointGroupNameMetadata` interface. _That's_ what's actually relevant here so I think your recommendation of referencing the interface instead of the class is a good one.